### DISCUSSION TITLE
API key not registering.

### DISCUSSION THREAD

**Discussion by Dingus6969** [reworkd/AgentGPT#25] [Q&A] [answered]
I have input my API key several times but it still ends up reaching the "this is a demo etc. etc." What can I do?

Also, is it able to save files somewhere online and give me access? Or email them through an smtp or save locally or something?

facepalm. I just realised you are working on getting it online and doing all that stuff haha I will watch this space!

Great work, by the way!

Thank you

**Comment by jasangill1** [comment, marked as answer]
Hello @Dingus6969 So sorry for the late reply we have recently made many changes that may have resolved this issue please feel free to try them out, and let us know if the problem still occurs !! currently, there isn't any file-saving integrated just output saving

**Reply by asim-shrestha** [reply]
Also, using your own API key should avoid that message in the future!
